src/typings/react-native-windows.d.ts

```diff
@@ -0,0 +1,36 @@
+/**
+* react-native-windows.d.ts
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* Type definition file for React Native Windows only components and modules
+* Definitions for extensions pertaining to existing React Native components are merged into the reaxt-native.d.ts file.
+*/
+
+declare module 'react-native-windows' {
+    //
+    // React
+    // ----------------------------------------------------------------------
+
+    import React = require('react');
+    import RN = require('react-native');
+
+    //
+    // Focusable view related declarations
+    // ----------------------------------------------------------------------
+    interface FocusableProps extends RN.ViewProps {
+        isTabStop?                      : boolean;
```

Comment:
>isTabStop?                      : boolean; [](start = 8, length = 42)

I think we prefer to avoid the column-like alignment because column-like alignment forces us to update the spacing of all the lines when we happen to add a new prop that has a name which is too long.

Reply:
react-native.d.ts is wildly inconsistent from your p.o.v. 
I'll follow your advice, hope I won't be getting the opposite as well...

---
In reply to: [159357801](https://github.com/Microsoft/reactxp/pull/404#discussion_r159357801) [](ancestors = 159357801)